src/main/java/com/psipher/application/api/GeneratePassword.java

```diff
@@ -0,0 +1,32 @@
+package com.psipher.application.api;
+
+import com.psipher.application.actions.GeneratePasswordOperations;
+import com.psipher.application.model.GeneratePasswordOutput;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GeneratePassword {
+    GeneratePasswordOperations generatePasswordOperations;
+
+    @Autowired
+    public GeneratePassword(GeneratePasswordOperations generatePasswordOperations) {
+        this.generatePasswordOperations = generatePasswordOperations;
+    }
+
+    @RequestMapping(path = "/generatepassword", method = RequestMethod.GET)
+    public GeneratePasswordOutput generatePassword(@RequestParam Integer length) {
+        GeneratePasswordOutput generatePasswordOutput = new GeneratePasswordOutput();
+        if (length != null && length >= 6) {
```

Comment:
Why are we allowing at 6 why cant me >=8 ? Let me know your views on it

Reply:
The usual minimum length for a password is 6, that's why I have selected it.